xcp_d/interfaces/workbench.py

```diff
@@ -816,3 +816,139 @@ class CiftiSeparateMetric(WBCommand):
     input_spec = _CiftiSeparateMetricInputSpec
     output_spec = _CiftiSeparateMetricOutputSpec
```

Comment:
do we still need the cifti separate metric?

Reply:
We could write one interface that covers all of `-cifti-separate` (including both `-metric` to get the surfaces and `-volume-all` to get the subcortical data), but I was feeling lazy and went with a separate interface.